script/Arcueid/Day 9/09_00_ARC09_4_2.txt

```diff
@@ -1041,5 +1041,5 @@ but Arcueid lowers her head, only for a moment,
 [sha:a5190732a2fc0fd06e9de1286592c0f072145730]{
 -- Page 304, Offset 22137.
 -- バイバイと手を振ってから、白い吸血鬼は夜の闇の中へ歩み去っていった。
-After waving her hand goodbye, the vampire in white walks away into the darkness of the night.
+After waving her hand goodbye?, the vampire in white walks away into the darkness of the night.
```

Comment:
Haven't heard this phrase before, is it correct?

Reply:
This exact formulation isn't really used I think. I'd say drop `her hand` to make
> After waving goodbye, the vampire...